File: src/main/java/com/itsci/mju/maebanjumpen/partyrole/service/AccountManagerService.java
package com.itsci.mju.maebanjumpen.partyrole.service;

import com.itsci.mju.maebanjumpen.partyrole.dto.AccountManagerDTO;

import java.util.List;

public interface AccountManagerService {
    List<AccountManagerDTO> getAllAccountManagers();
    AccountManagerDTO getAccountManagerById(int id);
    AccountManagerDTO saveAccountManager(AccountManagerDTO accountManager);
    void deleteAccountManager(int id);

    AccountManagerDTO updateAccountManager(int id, AccountManagerDTO accountManager);
}

File: src/main/kotlin/com/itsci/mju/maebanjumpen/partyrole/service/AccountManagerService.kt
package com.itsci.mju.maebanjumpen.partyrole.service

import com.itsci.mju.maebanjumpen.partyrole.dto.AccountManagerDTO

interface AccountManagerService {
    fun listAllAccountManagers(): List<AccountManagerDTO>
    fun getAccountManagerById(id: Int): AccountManagerDTO
    fun createAccountManager(accountManager: AccountManagerDTO): AccountManagerDTO
    fun deleteAccountManager(id: Int)
    fun updateAccountManager(id: Int, accountManager: AccountManagerDTO): AccountManagerDTO
}

